test/integration/basic/test/rendering.js

```diff
@@ -289,5 +289,10 @@ export default function ({ app }, suiteName, render, fetch) {
         expect($('#asPath').text()).toBe('Current asPath: /nav/with-hoc')
       })
     })
+
+    it('should show better error when undefined is thrown', async () => {
```

Comment:
"better" doesn't have any context outside of this PR

Reply:
Good point, updated